examples/galaxies/plot_demographics.py

```diff
@@ -0,0 +1,239 @@
+    return - np.log(1 - fenvironment) * phi_satellites
+
+
+# SkyPy amplitudes
+phic = schechter_smf_phi_centrals(phiblue, fsat)
+phis = schechter_smf_phi_satellites(phic, fsat)
+phimq = schechter_smf_phi_mass_quenched(phic, phis)
+phisq = schechter_smf_phi_satellite_quenched(phis, frho)
+
+
+# %%
+# Finally we simulate the galaxy populations.
+
+
+# Schechter mass functions
+z_centrals, m_centrals = schechter_smf(z_range, mstarb, phic, alphab, 1e9, 1e12, sky_area, cosmology)
+z_satellites, m_satellites = schechter_smf(z_range, mstarb, phis, alphab, 1e9, 1e12, sky_area, cosmology)
+z_massq, m_mass_quenched = schechter_smf(z_range, mstarb, phimq, alphab + 1, 1e9, 1e12, sky_area, cosmology)
+z_satq, m_satellite_quenched = schechter_smf(z_range, mstarb, phisq, alphab, 1e9, 1e12, sky_area, cosmology)
+
+
+logm_centrals = np.log10(m_centrals)
+logm_satellites = np.log10(m_satellites)
+logm_massq = np.log10(m_mass_quenched)
+logm_satq = np.log10(m_satellite_quenched)
+
+# log Mass bins
+bins = np.linspace(9, 12, 35)
+
+# Sky volume
+dV_dz = (cosmology.differential_comoving_volume(z_range) * sky_area).to_value('Mpc3')
+dV = np.trapz(dV_dz, z_range)
+dlm = (np.max(bins)-np.min(bins)) / (np.size(bins)-1)
+
+# log distribution of masses
+logphi_centrals = np.histogram(logm_centrals, bins=bins)[0] / dV / dlm
+logphi_satellites = np.histogram(logm_satellites, bins=bins)[0] / dV / dlm
+logphi_massq = np.histogram(logm_massq, bins=bins)[0] / dV / dlm
+logphi_satq = np.histogram(logm_satq, bins=bins)[0] / dV / dlm
+
+logphi_active = logphi_centrals + logphi_satellites
+logphi_passive = logphi_massq + logphi_satq
+logphi_total = logphi_active + logphi_passive
+
+
+# %%
+# Validation against SSD DR7 data
+# -------------------------------
+#
+# Here we compare our sampled galaxies and
+# we validate the model using the results from the best fit to
+# SDSS DR7 data in Weigel et al. (2016) [2].
+# The authors presented a comprehensive method to determine
+# stellar mass functions and apply it to samples in the local universe,
+# in particular to SDSS DR7 data in the redshift range from 0.02 to 0.06.
+#
+# Their data is presented as logarithmic distribution, therefore,
+# to perform the comparison we need to apply a conversion factor.
+# This factor allows us to go from :math:`\phi` to a :math:`\log \phi` plot
+# and compare with Weigel et al 2016 best-fit model. 
+
+
+# Load the data
+wtotal = Table.read('weigel16_total.csv', format='csv')
+wred = Table.read('weigel16_quiescent.csv', format='csv')
+wblue = Table.read('weigel16_active.csv', format='csv')
+wcentral = Table.read('weigel16_central.csv', format='csv')
+wsatellite = Table.read('weigel16_satellite.csv', format='csv')
+
+# %%
+
+
+# Plot
+fig, (ax1, ax2, ax3) = plt.subplots(nrows=1, ncols=3, figsize=(16, 6), sharex=True, sharey=True)
+fig.suptitle('Galaxy Demographics Simulation', fontsize=26)
+
+ax1.plot(wblue['logm'], 10**wblue['logphi'], color='k', label='Weigel+16 active', lw=1)
```

Comment:
The Weigel+ lines for the passive and the total sample include errors. Does the blue sample not have errors?